datalake/processing_stack.py

```diff
@@ -184,6 +199,7 @@ def __init__(self, scope: core.Construct, stack_id: str, deploy_env, vpc, **kwar
                         ),
                     ),
                 )
+                assets_bucket.grant_read_write(lambda_function)
```

Comment:
This gives write access to *all* the lambdas. We should only give write access to the content parser.